lib/hypernova/controller_helpers.rb

```diff
@@ -88,7 +88,7 @@ def hypernova_batch_after
           result = @hypernova_batch.submit!
           on_success(result, hash)
         rescue StandardError => e
-          on_error(e)
+          on_error(e, {}, hash)
```

Comment:
btw, also realized that I need to modify [this](https://github.com/airbnb/hypernova-ruby/blob/e6c6553a4838203f4cdd62368ba1c146f632233d/lib/hypernova/plugin_helper.rb#L51)